demo/actuator_demo/tkinter_client/tkinter_traffic_sign.py

```diff
@@ -0,0 +1,83 @@
+import tkinter
+from tkinter import *
```

Comment:
Any reason why using tk and not Qt?

Reply:
Mainly because I don’t know how to use QT, and Tkinter seemed more easy to me :D . Also, it’s very likely that this code will only be used for the demo.

Reply:
np. Welcome on board: never used QT. :D
tk is fine as well, and it's a very simple code.